components/automate-ui/src/app/modules/infra-proxy/edit-environment-attribute-modal/edit-environment-attribute-modal.component.ts

```diff
@@ -191,57 +191,48 @@ export class EditEnvironmentAttributeModalComponent implements OnChanges, OnInit
     this.updateSuccessful = false;
     this.updateInProgress = true;
 
-    if (this.label === 'Constraints') {
```

Comment:
Author note: repeated blocks of code is a code smell -- not only does the reworked block of code eliminate the repeated lines, but by doing so, you can instantly see what is different about the three cases. That is much harder to discern in the original.